lib/dfe/analytics.rb

```diff
@@ -127,6 +128,12 @@ def self.custom_events
       []
     end
 
+    def self.event_debug
```

Comment:
Maybe `event_debug_filters`, so it's clear that this isn't a switch?

Reply:
Fixed.